numpy/__init__.py

```diff
@@ -357,6 +357,26 @@ def _mac_os_check():
                 raise RuntimeError(msg)
     del _mac_os_check
 
+    def _win_os_check():
+        """
+        Quick Sanity check for Windows OS: look for fmod bug issue 16744.
+        """
+        try:
+            a = arange(13 * 13, dtype= float64).reshape(13, 13)
+            a = a % 17  # calls fmod
+            linalg.eig(a)
+        except Exception:
+            msg = ("The current Numpy installation ({!r}) fails to "
+                   "pass a sanity check due to a bug in the windows runtime. "
+                   "See this issue for more information "
+                   "https://developercommunity.visualstudio.com/content/problem/1208774/fpu-exception-in-fmod0-x-in-windows-10-version-200.html")
```

Comment:
2004, not 200?

Reply:
The URL is correct, despite missing the 4.

Reply:
NVM.

Reply:
I used "https://tinyurl.com/y3dm3h86" instead.